depends/patches/qt/drop_lrelease_dependency.patch

```diff
@@ -0,0 +1,20 @@
+commit 67b3ed7406e1d0762188dbad2c44a06824ba0778
+Author: fanquake <fanquake@gmail.com>
+Date:   Tue Aug 18 15:24:01 2020 +0800
+
+    Drop dependency on lrelease
+
+    Qts buildsystem insists on using the installed lrelease, but gets
+    confused about how to find it. Since we manually control the build
+    order, just drop the dependency.
```

Comment:
I believe this is/was only necessary because of the funky way we build qt to avoid building a bunch of unneeded stuff. If we ever move to a simple 'make' for qt (rather than the 4 we do now), it should be able to handle all of this for us.

That may be worth investigating as a follow-up, it's been a long time since we've revisited it. Maybe qt's build has reduced unnecessary compiles since then.

Reply:
A revisit into how we are building Qt will happen as part of #19716 (which is also why I wanted to keep this a sed -> patch migration with no additional changes). I think there's a good chance a bunch of complexity is going to disappear.